pkg/kubelet/apis/resourcemetrics/v1alpha1/config.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	summary "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+	"k8s.io/kubernetes/pkg/kubelet/server/stats"
+)
+
+// Version is the string representation of the version of this configuration
+const Version = "v1alpha1"
+
+// Config is the v1alpha1 resource metrics definition
+func Config() stats.ResourceMetricsConfig {
+	return stats.ResourceMetricsConfig{
+		NodeMetrics: []stats.NodeResourceMetric{
+			{
+				Name:        "node_cpu_usage_seconds_total",
+				Description: "Cumulative cpu time consumed by the node in core-seconds",
+				ValueFn: func(s summary.NodeStats) (*float64, time.Time) {
```

Comment:
Since we want to remove summary endpoint at last, should we use our own struct rather than summay-sytle struct?
I think maybe we need summary endpoint and resource-metrics endpoint co-exist for 1~2 versions. 
So maybe split them totally is a better solution?

Reply:
We could, but that would mean having two different backing implementations of this API, which would copy much of the code used to generate the summary API.  We have one implementation for DockerShim, one for CRI.

This implementation doesn't prevent us from removing the summary API, as that just means not exposing the `/stats/summary` endpoint.  After we remove that endpoint entirely, we can easily refactor this implementation to not use the summary API anymore.  For now, it is easier to maintain in this state, as we don't have to duplicate code for dealing with multiple metric sources.